common/config/rush/pnpm-lock.yaml

```diff
@@ -946,7 +947,7 @@ packages:
   /@types/body-parser/1.19.0:
     dependencies:
       '@types/connect': 3.4.34
-      '@types/node': 8.10.66
```

Comment:
Any idea why `@types/node` keeps going to version 10.x instead of 8.x?

Reply:
I should revert this.  I have no ideas. It's from `rush update`

Reply:
Reverted. Still a mystery to me. I did notice that before https://github.com/Azure/azure-sdk-for-js/pull/13999/files there were some v10 as well in the lock file.